Description: Backport upstream fix for tests with incorrect locale name
Origin: upstream

Index: php/ext/standard/tests/strings/strtolower.phpt
===================================================================
--- php.orig/ext/standard/tests/strings/strtolower.phpt
+++ php/ext/standard/tests/strings/strtolower.phpt
@@ -22,7 +22,7 @@ if( substr(PHP_OS, 0, 3) == 'WIN') {
 if( substr(PHP_OS, 0, 3) == 'WIN') {
   setlocale(LC_ALL, 'C');
 } else {
-  setlocale(LC_ALL, 'en-US.UTF-8');
+  setlocale(LC_ALL, 'en_US.UTF-8');
 }
 
 echo "*** Testing strtolower() with all 256 chars ***\n";
Index: php/ext/standard/tests/strings/strtoupper1.phpt
===================================================================
--- php.orig/ext/standard/tests/strings/strtoupper1.phpt
+++ php/ext/standard/tests/strings/strtoupper1.phpt
@@ -22,7 +22,7 @@ if( substr(PHP_OS, 0, 3) == 'WIN') {
 if( substr(PHP_OS, 0, 3) == 'WIN') {
   setlocale(LC_ALL, 'C');
 } else {
-  setlocale(LC_ALL, 'en-US.UTF-8');
+  setlocale(LC_ALL, 'en_US.UTF-8');
 }
 
 echo "*** Testing strtoupper() with all 256 chars ***\n";
